cli/src/commands/generate/generate.ts

```diff
@@ -26,12 +25,12 @@ function loadFile(path: string): any {
 }
 
 
-function instantiateAdditionalTopLevelProperties(pattern: any, schemaDirectory: SchemaDirectory): any {
-    const properties = pattern?.properties;
-    if (!properties) {
+function instantiateAdditionalTopLevelProperties(pattern: object, schemaDirectory: SchemaDirectory): object {
```

Comment:
using object over any is an improvement, but we should be able to do better. The result of this function should be a valid CALMInstantiation shouldn't it?

Reply:
removing this function entirely